pymc/distributions/shape_utils.py

```diff
@@ -565,26 +569,30 @@ def resize_from_observed(
     return resize_shape, observed
 
 
-def find_size(shape=None, size=None, ndim_supp=None):
+def find_size(
+    shape: Optional[WeakShape],
+    size: Optional[StrongSize],
+    ndim_supp: int,
+) -> Tuple[Optional[StrongSize], Optional[int], Optional[int], int]:
     """Determines the size keyword argument for creating a Distribution.
 
     Parameters
     ----------
-    shape : tuple
+    shape
         A tuple specifying the final shape of a distribution
-    size : tuple
+    size
```

Comment:
These hints were inaccurate and I didn't want to add the fully expanded meaning of `Optional[WeakShape]` etc.